docs/architecture/readMe.md

```diff
@@ -0,0 +1,16 @@
+# Architecture Decision Records
```

Comment:
minor: I think the standard casing of `readme` is either `readme` or `README` from what I have seen on GitHub? I believe in this repo we're using `README`, maybe align on that?

Reply:
Renamed.